src/app/[lng]/(refuel)/refuel/page.tsx

```diff
@@ -1,9 +1,25 @@
-import dynamic from 'next/dynamic';
+import { cookies } from 'next/headers';
+import App from 'src/app/ui/app/App';
+import type { ThemeModesSupported } from 'src/types/settings';
 
-const App = dynamic(() => import('../../../ui/app/App'), { ssr: false });
+export const dynamic = 'force-dynamic';
```

Comment:
What is this for? 👀

Reply:
@chybisov Thanks for asking! I ran into a weird issue after making use of cookies within those "page.tsx" files.
It ran nice with next dev, but after doing the builld, I was not able to run the page without "DYNAMIC_SERVER_USAGE" error.

Reply:
I have removed it in most places by adding those pages as an async function. One spot though, still uses it